doc/release-notes-19133.md

```diff
@@ -0,0 +1,7 @@
+## CLI
+
+A new `bitcoin-cli -generate` command, equivalent to RPC `generatenewaddress`
```

Comment:
drop 19357 unless we add 19133

Reply:
yeah, it's weird to have release notes for smth not yet implemented